packages/ra-ui-materialui/src/input/ArrayInput.tsx

```diff
@@ -85,6 +86,8 @@ export const ArrayInput = ({
                     isRequired={isRequired(validate)}
                 />
             </InputLabel>
+            {helperText &&
```

Comment:
We might have validation applied to the `ArrayInput` itself, in which case the `HelperText` should display the error instead of the helper text like the other inputs do. You can find a very close implementation in `SelectArrayInput`.

Reply:
Updated, however I am not sure of the props useInput requires, do all inputs receive them?